test/e2e/util/csi/common.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2020 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	snapshotterClientSet "github.com/kubernetes-csi/external-snapshotter/client/v4/clientset/versioned"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/vmware-tanzu/velero/test/e2e/util/k8s"
+)
+
+func GetClients() (*kubernetes.Clientset, *snapshotterClientSet.Clientset, error) {
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
+	configOverrides := &clientcmd.ConfigOverrides{}
+	kubeConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loadingRules, configOverrides)
+	clientConfig, err := kubeConfig.ClientConfig()
+	if err != nil {
+		return nil, nil, errors.WithStack(err)
+	}
+
+	client, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		return nil, nil, errors.WithStack(err)
+	}
+
+	snapshotterClient, err := snapshotterClientSet.NewForConfig(clientConfig)
+	if err != nil {
+		return nil, nil, errors.WithStack(err)
+	}
+
+	return client, snapshotterClient, nil
+}
+
+func GetVolumeSnapshotContentNameByPod(client TestClient, podName, namespace, backupName string) (string, error) {
+	pvcList, err := GetPvcByPodName(context.Background(), namespace, podName)
+	if err != nil {
+		return "", err
+	}
+	if len(pvcList) != 1 {
```

Comment:
It's possible one Pod has more than one PV, but please confirm whether this condition is acceptable for test environment.